msg/MissionProgress.msg

```diff
@@ -0,0 +1,3 @@
+builtin_interfaces/Time time_stamp # Timestamp from when the message was send
+string sender_id # Name of the node that send this command
+float32 progress # Progress (0..1.0)
```

Comment:
Maybe change comment to "#Progress (0 ... 1.0)" to increase legibility, I was confused here for a second.

Reply:
Changed to `(0 .. 1.0)`: https://github.com/THI-Drone/interfaces/pull/3/commits/3a7b4f248c2a4cf34bb4702476930c6d5eaf1023